### PR TITLE
[Security] Upgrade packages

### PR DESCRIPTION
**Pull request by palmenhq**
It's _really_ about time..

### CODE REVIEW

File: possibleGraphqlTypes.json [hunk 1]
```diff
@@ -0,0 +1,241 @@
+{
```
Comment (gustaveen) [inline possibleGraphqlTypes.json:1]: The name possible is a bit confusing to me :) Can't it just be named `graphqlTypes.json`?

**Comment by gustaveen** [comment]
Gold star to you for doing this! ⭐️

**Comment by styrelius** [comment]
Agreed - so awesome that you're doing this!! 👑

File: possibleGraphqlTypes.json (hunk 1 above)
Comment (palmenhq) [reply possibleGraphqlTypes.json:1]: Unsure how to treat this naming.. this is how they want us to use it https://github.com/HedvigInsurance/web-onboarding/pull/382/files#diff-56db900dae2350588c5403a48a41390c684cdf3b11a3ad41fdc7113aaee55fd6R6-R35

File: codegen.yml [hunk 2]
```diff
@@ -2,7 +2,7 @@ overwrite: true
 schema: 'https://graphql.dev.hedvigit.com/graphql'
 documents: 'src/**/*.graphql'
 generates:
-  src/data/graphql.tsx:
+  src/client/data/graphql.tsx:
```
Comment (gustaveen) [inline codegen.yml:5]: Ah nice, just changed this locally :)

**Review by gustaveen — APPROVED** [review]

**Pull request merged**